stripe/util.py

```diff
@@ -260,6 +260,15 @@ def convert_to_stripe_object(
         return resp
 
 
+def convert_to_dict(obj):
+    if isinstance(obj, list):
+        return [convert_to_dict(i) for i in obj]
+    elif isinstance(obj, dict):
```

Comment:
Would you mind adding a comment here that this will work on a `StripeObject` because it will advertise itself as a `dict` and allow itself to be iterated (I think that's the reason anyway).

Reply:
👍